### ISSUE TITLE
Loss not decrease

### ISSUE THREAD

**Issue by zhaozj89** [lengstrom/fast-style-transfer#88]
Hi, just want to ask a question. I use batch_size=10, and have trained using a style image for about 1800  iterations, with all COCO train2014 data. But the loss seems not go down, especially the style loss (as shown below). Can you suggest me what I am doing wrong. Thanks!

![qq 20170724235923](https://user-images.githubusercontent.com/5336612/28555198-ccd3e486-7130-11e7-8b68-136447a6b371.png)

**Comment by mdramos** [comment]
I was experiencing buggy behavior with the most recent commits (even loss returning `nan` - did not debug), I reverted to pre-5 days ago and results were better.

**Comment by zhaozj89** [comment]
I also notice the style layers are changed https://github.com/lengstrom/fast-style-transfer/commit/aab9bf71bf47ff8a5cd78dd01972ea9f9b0e7ae0, which is different from the original paper http://cs.stanford.edu/people/jcjohns/papers/eccv16/JohnsonECCV16.pdf.

**Comment by lengstrom** [comment]
I'm still investigating buggy behavior in the current impl. As Michael
said, reverting to 5 days ago is probably the best plan of action for now
and I will update the repo as such tonight.

The style loss layers have always been different from
the original paper (for a reason). I changed them temporarily but
immediately changed them back.

I think for the last few days I've forgotten that many people use my
software and that this project is no longer just an experiment of mine. I'm
sorry for making changes so rashly.

**Comment by mdramos** [comment]
@lengstrom good points and ty, also please consider versioning and Feature Branch Workflow i.e. `master->dev->feature`.

**Comment by lengstrom** [comment]
Thanks for the pointer - that's what I traditionally have done! I think I
got a bit overconfident in the last week :)

**Comment by slpal** [comment]
has this bug been fixed? 
I tried to train the net like this "python style.py --style examples/style/rain_princess.jpg --checkpoint-dir rain_princess_checkpoint --test examples/content/chicago.jpg --test-dir rain_princess_test/ --learning-rate 1e-3 --checkpoint-iterations 1000 --batch-size 4 --epochs 7"
but get loss = nan

**Comment by lengstrom** [comment]
reverted to working commit
